packages-exp/performance-exp/src/services/oob_resources_service.ts

```diff
@@ -14,36 +14,44 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import { Api } from './api_service';
 import { Trace } from '../resources/trace';
 import { createNetworkRequestEntry } from '../resources/network_request';
 import { TRACE_MEASURE_PREFIX } from '../constants';
 import { getIid } from './iid_service';
+import { PerformanceController } from '../controllers/perf';
 
 const FID_WAIT_TIME_MS = 5000;
 
-export function setupOobResources(): void {
+export function setupOobResources(performance: PerformanceController): void {
```

Comment:
Can we name the PerformanceController instance performanceController for better readability?

Reply:
Done.